practical-penguins/trivia_tavern/twilio_messenger/models.py

```diff
@@ -0,0 +1,43 @@
+from typing import Tuple, List, Optional
```

Comment:
well made. great code quality. docstrings and type annotations makes it easier to read.

Reply:
Only minor nit here is that docstrings should generally follow the format defined in [PEP 257](https://www.python.org/dev/peps/pep-0257/). But that can sometimes vary a bit based on your team's defined conventions, so I don't consider it a huge issue. I'm much more concerned about the presence and content of docstrings in general rather than the exact format they follow.